### PR TITLE
Initial namespace conversion

### PR DESCRIPTION
**Pull request by smillidge**
Signed-off-by: Steve Millidge <steve.millidge@payara.fish>

### CODE REVIEW

File: specification/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <groupId>jakarta.platform</groupId>
     <artifactId>platform-spec</artifactId>
     <packaging>pom</packaging>
-    <version>9-SNAPSHOT</version>
+    <version>9.0-SNAPSHOT</version>
```
Comment (kwsutter) [inline specification/pom.xml]: We decided with Jakarta EE 8 that the platform specification (and web profile) will use a single digit for the release version.  Thus, this should stay at 9-SNAPSHOT.

**Comment by kwsutter** [comment]
This looks quite complete, @smillidge!  Could we just revert that pom update and then we could go ahead with approval and merge.  Thanks!

**Comment by kwsutter** [comment]
> This looks good, but you need to verify that any APIs that use property names starting with
> "javax" are actually changing the property names to start with "jakarta". I would expect them
> to, but some may not have been thinking about this issue.
> 
> (And of course this is another issue to be dealt with by any backward compatibility solution.)

Good point, @bshannon.  Actually, I would expect most properties to stay with the same javax prefix.

**Comment by bshannon** [comment]
> Actually, I would expect most properties to stay with the original java or javax prefix.

Really? Why is that?  Sounds like an issue we should discuss and put out a recommendation on.

> I don't think we have to know the "final answer" before merging these changes. Maybe we can create a separate issue to come back and verify the property names before we go with the final version of the Specs...

There were a few instances of property names being changed, I believe.  We don't need to know the
final answer as long as we remember to update this as necessary once the final answer is known.

**Comment by scottmarlow** [comment]
> There were a few instances of property names being changed, I believe.

[One example of a property change to jakarta is in javadoc here.](https://github.com/eclipse-ee4j/jpa-api/blob/master/api/src/main/java/jakarta/persistence/CacheStoreMode.java#L21)

**Comment by kwsutter** [comment]
> > Actually, I would expect most properties to stay with the original java or javax prefix.
> 
> Really? Why is that? Sounds like an issue we should discuss and put out a recommendation on.

When the required javax->jakarta name change was being discussed, the property names did not have to change.  They could be changed, but there was no requirement.  But, I also agree that this item should be discussed.  I'll add it to the Platform agenda for today.

**Comment by bshannon** [comment]
The property names were discussed and approved so I think we're ready to update this.

**Comment by smillidge** [comment]
OK @kwsutter I reverted the pom change

**Pull request merged**